velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -580,30 +614,39 @@ class ExpressionFuzzer {
       }
     }
 
+    // Store data and expression in case of reproduction.
+    std::shared_ptr<BaseVector> copiedResult;
+    std::string sql;
+    // Deep copy to preserve the initial state of result vector.
+    if (!FLAGS_repro_persist_path.empty()) {
+      copiedResult =
+          resultVector ? BaseVector::copy(*resultVector) : resultVector;
+      sql = plan->toString();
```

Comment:
Shouldn't this call plan->toSql() ?

Reply:
I think for ITypedExpr there is no toSql() call. The equivalent is just toString() if I'm not wrong.

Reply:
This was what I did before. I think somehow after compilation or execution the expressions got evaluated already and is no long the original structure.

Reply:
Let me see what I can do here.

Reply:
Ah. The reason it does not preserve original expression when using exprSet is because it enables constant folding. Disabling it makes it work. I will make the changes.